cordova-fetch/spec/package.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
I just copy it into temporary tests folder so I can test - - save. Any alternative suggestion? Should I store it somewhere else?